src/components/App/index.tsx

```diff
@@ -1,8 +1,9 @@
 import React from "react";
+import "./styles.scss";
+import PhotoViewer from "../PhotoViewer/PhotoViewer";
```

Comment:
can we remove this import now it isn't being used here? thanks :)

Reply:
that's done!